### DISCUSSION TITLE
Default SBOM Generate

### DISCUSSION THREAD

**Discussion by heheda1212138** [webpack/webpack#17912] [Q&A]
## Feature request

**What is the expected behavior?**
Is Webpack going to generate SBOM by default in the future without any plugin?
**What is motivation or use case for adding/changing the behavior?**
FE developer can generate SBOM without any plugin, and SBOM may be the default option in the future FE environment.
**How should this be implemented in your opinion?**
Control whether generate SBOM list by an option, and generate SBOM in the product code.
**Are you willing to work on this yourself?**
no

**Comment by alexander-akait** [comment]
Hello, what is SBOM?
